echopype/convert/set_groups_azfp.py

```diff
@@ -110,12 +110,12 @@ def set_beam(self) -> xr.Dataset:
             )
 
         # Largest number of counts along the range dimension among the different channels
-        longest_range_bin = np.max(unpacked_data["num_bins"])
-        range_bin = np.arange(longest_range_bin)
+        longest_range_sample = np.max(unpacked_data["num_bins"])
+        range_sample = np.arange(longest_range_sample)
 
         # Pad power data
-        if any(unpacked_data["num_bins"] != longest_range_bin):
-            N_tmp = np.full((len(N), len(ping_time), longest_range_bin), np.nan)
+        if any(unpacked_data["num_bins"] != longest_range_sample):
+            N_tmp = np.full((len(N), len(ping_time), longest_range_sample), np.nan)
```

Comment:
Just as a note-- @b-reyes : these can benefit from what you have tested some time ago re the padding. Also all the padding can be consolidated in a single place as we have discussed.